chisel/src/session_source.rs

```diff
@@ -480,7 +481,7 @@ import {{Cheats}} from "forge-std/Vm.sol";
 {}
 
 contract {} {{
-    Cheats internal constant vm = Cheats(address(uint160(uint256(keccak256("hevm cheat code")))));
+    Cheats internal constant vm = Cheats(address(uint176(uint256(sha3("hevm cheat code")))));
```

Comment:
this place is a little problem. We cannot just use code above because we need to build ICAN address with checksum and prefix. What we can do is to create a const `sha3("hevm cheat code")` in Rust and pass a built address inside string. But in which format address should be so Ylem can handle it? Could it be something like `Cheats(address("ab37...12"))` or smth like this?